core/lib/pipeline/object-manipulator.ts

```diff
@@ -46,12 +50,32 @@ export async function manipulateObject(
     if (ast === undefined) {
       throw new Error('Cannot remove root node.');
     }
-    const newObject = transformer(doc, Clone(hit.value), hit.path);
-    const newAst = newObject === undefined
-      ? undefined
-      : ToAst(newObject); // <- can extend ToAst to also take an "ambient" object with AST, in order to create anchor refs for existing stuff!
-    const oldAst = ResolveRelativeNode(ast, ast, hit.path);
-    ast = ReplaceNode(ast, oldAst, newAst) || (() => { throw new Error('Cannot remove root node.'); })();
+
+    try {
+      const newObject = transformer(doc, Clone(hit.value), hit.path);
+      const newAst = newObject === undefined
+        ? undefined
+        : ToAst(newObject); // <- can extend ToAst to also take an "ambient" object with AST, in order to create anchor refs for existing stuff!
+      const oldAst = ResolveRelativeNode(ast, ast, hit.path);
+      ast = ReplaceNode(ast, oldAst, newAst) || (() => { throw new Error('Cannot remove root node.'); })();
+    } catch (err) {
+      // Background: it can happen that one transformation fails but the others are still valid. One typical use case is
+      // the common parameters versus normal HTTP operations. They are on the same level in the path, so the commonly used
+      // '$.paths.*.*' "where selection" finds both, however, most probably the transformation should and can be executed
+      // either on the parameters or on the HTTP operations, i.e. one of the transformations will fail.
+      if (config != null) {
+        let errorText = `Directive with 'where' clause '${whereJsonQuery}' failed by path '${hit.path}`;
+        if (extraTransforationContext != null) {
```

Comment:
You'll need to update references of `extraTransforationContext` to `transformationString` here and in the error text below.  Also, now that I look at this again `transformString` is probably the right name, do you mind changing it one more time?

Reply:
Ah, for sure :) I just used the Github feature to accept your recommendation, and dummily have to changed the other part of the code. Sorry for that and thank you for spotting it out. Interestingly, my local compilation did not complain that something is wrong with the name, this might a TypeScript/Javascript thing that I have to learn.

Reply:
By the way, this was my original naming as well. I changed it `extraTransformationContect` to avoid the confustion that it is _not_ used in the transformation, the `transformer` lambda does the job. I am happy with the `transformationString`, just meant to give you background of it.

Reply:
Sounds good!